### PR TITLE
DEI CFP: Astropy in Outreachy

### PR DESCRIPTION
**Pull request by bsipocz**
cc @mirca, @dpshelio, and @Cadair 

This proposal is to participate in a mentorship partnering with an organization expert in recruiting talent from groups underrepresented in tech. There are multiple good precedents of science FOSS projects participating with whom the proposing team has good connections to share experiences.

### CODE REVIEW

File: proposals/DEI/DEI_Astropy_Outreachy.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+##Impact
+
+As it was highlighted in the call for proposal the Astropy community is extremely biased in many 
+axes. This proposal combines the original mentoring intent of the funding in the Moore grant with
+Diversity and Inclusion aspirations.
+Outreachy has an excellent track record to tap into talent pools that none of the usual Astropy
+efforts reach. The internship provides a long coding mentorship for people from underrepresented 
+groups, as well as some mentoring to gain familiarity with open source workflows for the people
+participating in the earlier, and shorter contributing phase Outreachy.
```
Comment (mirca) [inline proposals/DEI/DEI_Astropy_Outreachy.md:31]: contributing phase Outreachy --> contributing phase ``of`` Outreachy?

**Comment by bsipocz** [comment]
We might need a decision about this sooner rather than later as Outreachy is in it's initial application phase for the global South (with coding period between Dec-March).

I've pinged them to ask whether application is still open for FOSS projects as I hasn't notice a separate call for organizations to pitch interest.

**Comment by astrofrog** [comment]
I’ve heard a lot of good things about Outreachy (in particular from Jupyter) and it is significantly different from GSoC (for a start not just students). It has the benefit of being an already up and running scheme. I think it would be really worthwhile to try once to see how it goes (in addition to different approaches) 👍

**Comment by bsipocz** [comment]
I'll report back once I hear something from them about the org timelines and checklists. 

I also heard very good things about it from multiple projects, projects that did participated in GSoC, too. (And David has even more contacts who mentored for Outreachy, and I believe it might even came up last year at the gsoc summit, but sadly I haven't attended that)

**Comment by hamogu** [comment]
Based on the community feedback, this project was selected for funding as announced in an email from the coordination committee. Is there anything that the finance committee needs to do to get this moving before the outreachy deadline?

**Comment by bsipocz** [comment]
So far I only needed to state on the Outreachy side that the funding is secured, I suppose more details will follow later when an intern match is reached. 
I'll send out a project idea solicitation to the maintainers later this afternoon.

**Pull request merged**